observability/interceptor.go

```diff
@@ -0,0 +1,96 @@
+package observability
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/alecthomas/errors"
+	"github.com/bufbuild/connect-go"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+
+	"github.com/TBD54566975/ftl/internal/log"
+)
+
+const ftlDirectRoutingHeader = "FTL-Direct"
+const ftlVerbHeader = "FTL-Verb"
```

Comment:
@alecthomas let's chat about this one when you have a sec. I doubt we want to duplicate this here, but I can't import rpc because of a circular dep.

Also want to confirm that we want to find the verb ref via headers in the context.

Reply:
Mmm yeah I see. I think this is okay for now. You can remove the ftlDirectRoutingHeader though?